### PR TITLE
French & Turkish translations for fetch4fd. Some licensing information would have been useful.

### PR DESCRIPTION
**Pull request by cardpuncher**

### CODE REVIEW

File: fetch4fd/nls/fetch4fd.fr-UTF-8 [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{Mémoire : }
+{Mo RAM}
+{Disques installés : }
+{libre}
+{Version de DOS rapportée : }
```
Comment (andrewbird) [inline fetch4fd/nls/fetch4fd.fr-UTF-8:5]: Odd encoding for a UTF-8 file here?

**Comment by cardpuncher** [comment]
On 23.03.2024 14:46, Andrew Bird wrote:
> ***@***.**** commented on this pull request.

> Odd encoding for a UTF-8 file here?

When I take a look at 
<https://github.com/cardpuncher/fd-nls/blob/master/fetch4fd/nls/fetch4fd.fr-UTF-8> 
everything seems fine, but not on the PR. The Turkish UTF-8 file also 
has problems, so I'm going to send an update to the pull request. Thanks 
for catching this.

**Comment by shidel** [comment]
> Some licensing information would have been useful.

Yeah. Laaca does not have a landing page for the program and I did not see any information regarding licensing. To me, ithat means it is just Free w/sources. As such, it cannot be considered for inclusion on the OS release media at present.

**Comment by shidel** [comment]
@andrewbird, good to merge now?

File: fetch4fd/nls/fetch4fd.fr-UTF-8 (hunk 1 above)
Comment (andrewbird) [reply fetch4fd/nls/fetch4fd.fr-UTF-8:5]: So I'm really confused  now, so maybe it's some github thing. The changes in the PR overview https://github.com/shidel/fd-nls/pull/187/files still look strange. To be sure I checked out @cardpuncher 's branch to my local machine and they look fine.

**Comment by cardpuncher** [comment]
On 23.03.2024 15:36, shidel wrote:

> Yeah. Laaca does not have a landing page for the program and I did not 
> see any information regarding licensing. To me, ithat means it is just 
> Free w/sources. As such, it cannot be considered for inclusion on the OS 
> release media at present.

Hey @shidel I sent a message to Laaca regarding this, and here's his answer:
"No, it is GNU/GPL.
Just run "Mysysinf /?" and you will see it. Or look to any language 
message file like MYSYSINF.EN or similar. It is also there."

Which is great news, maybe Mysysinf could be included in the release media?

**Pull request merged**